galsim_extra/mixed_scene.py

```diff
@@ -58,8 +59,23 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
         # object ends up being chosen.
         ignore = ignore + ['objects', 'magnify', 'shear', 'obj_type']
 
+        stamp_xsize, stamp_ysize, image_pos, world_pos = super(MixedSceneBuilder, self).setup(config,base,xsize,ysize,ignore,logger)
+        # option to shear the full scene.
+        if config.get('shear_full_scene', True):            
+            shear = galsim.Shear(g1=float(base["stamp"]["shear"]["g1"]), g2=float(base["stamp"]["shear"]["g2"]))
```

Comment:
`base["stamp"]` is aka `config` in this context.
But again, don't use direct access with `["g1"]` etc.  Use ParseValue since they may not be native floats.  They may be dicts that generate floats.

Reply:
```
if shear_scene:       
            g = galsim.config.ParseValue(config, 'shear', base, float)[0]
            shear = galsim.Shear(g1=g["g1"], g2=g["g2"])
```
Does this seem like a way? 'shear' key has three values ('type', 'g1' and 'g2') so I wasn't sure if I can use galsim.config.ParseValue for more than one key...

Reply:
No.  Just
`shear = galsim.config.ParseValue(config, 'shear', base, float)[0]`